src/universal/components/Action/Action.js

```diff
@@ -60,7 +49,7 @@ const Action = () => {
         <AsyncRoute isPrivate path="/welcome" mod={welcome} />
         <AsyncRoute path="/admin/graphql" mod={graphql} />
         <AsyncRoute path="/admin/impersonate/:newUserId" mod={impersonate} />
-        <AsyncRoute path="/invitation/:id" mod={invitation} />
+        <AsyncRoute isPrivate path="/invitation/:inviteToken" mod={invitation} />
```

Comment:
Is this right? Should this be isPrivate?

Reply:
yeah, the way new auth was designed, all the redirects happen in the same window. Before, when you weren't logged in, you'd see a modal asking you to log in. Now, the only way to log in is via the /signup route, so i just redirect there.